### PR TITLE
Timing

### PR DESCRIPTION
**Pull request by h-lame**
To address #26.

I don't think there's anything contentious in here, but wouldn't mind another set of eyes to prune my verbiage if required.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -99,11 +101,19 @@ If you are using the projector you should favour large fonts and a high contrast
...
 #### Timing
 
-Our normal pattern for a meeting is two speakers with 40 minute time slots, although one-off meetings might be different. For example, our February meetings are for lightning talks of no more than 10 minutes in length.
+Our meetings are 1 hour 30 minutes long, with about 5-10 minutes reserved at the start for admin and announcements from the group. To make it simpler to organise filling the remining 80 minutes we offer three standard slot sizes for talks:
+
+* 40 minutes
+* 25 minutes
+* 10 minutes
+
+We combine these different slots to fill the meeting; for examplebe two 40s, three 25s, or one of each. On rare occasions we use a single 80 minute slot for a panel discussion or practical event. Our February "lightning talk" meetings are always made up of eight 10 minute slots.
```
Comment (tomstuart) [inline index.md]: Do you mean `examplebe`? I don’t think you mean `examplebe`.

**Comment by tomstuart** [comment]
This looks like a good improvement to me. We can always iterate if people have suggestions.

File: index.md (hunk 1 above)
Comment (h-lame) [inline index.md]: One time I'd like to write something without a typpo.

**Comment by h-lame** [comment]
Could've sworn I merged this already.

**Pull request merged**